### PR TITLE
[Feature] - Dashboard Tools Page

### PR DESCRIPTION
**Pull request by mkanoria**
## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)

## Implementation description
* Branched off of @christineNg1028 work on the Tool component
* Rebased to get Daniel & Tony's work on Modal and styling

## Steps to test
1. Run `yarn dev` and go to `http://localhost:3000/admin/dashboard/tools`

## What should reviewers focus on?
* 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

image

image

image

> Need to merge this after Tony -> Daniel -> Christine

### CODE REVIEW

File: data/tools.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const toolsList = [
```
Comment (mkanoria) [inline data/tools.ts:1]: Review this^

File: pages/admin/dashboard/tools.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
+import React, { useState, useEffect } from "react";
```
Comment (mkanoria) [inline pages/admin/dashboard/tools.tsx:1]: Review this^

File: src/components/toolCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react";
```
Comment (mkanoria) [inline src/components/toolCard/index.tsx:1]: This has a couple changes compared to christine's PR

File: pages/admin/dashboard/tools.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+    const [selectedTool, setSelectedTool] = useState("");
+    // Modal can be of "publish" or "delete" mode
+    const [modalMode, setModalMode] = useState("");
+    const publishConfirmation = `Are you sure you want to delete ${selectedTool}? Your tool will be available to the public!`;
+    const deleteConfirmation = `Are you sure you want to publish ${selectedTool}? This is a permanent action that cannot be undone.`;
+
+    const [selectedTab, setSelectedTab] = useState("draft");
+
+    // TODO: Need to update this to calculate relative date
```
Comment (tawnyzhao) [inline pages/admin/dashboard/tools.tsx:27]: I implemented relative date on my branch

**Review by tawnyzhao — APPROVED** [review]

**Review by christineNg1028 — APPROVED** [review]

File: src/components/modal/index.tsx [hunk 5]
```diff
@@ -60,20 +61,14 @@ export const Modal: React.FC<ModalProps> = (props) => {
...
                         </Button>
                         {!alignButtonsRight && <Spacer />}
-                        {/* variant="solid" to apply colorScheme */}
-                        <Button
-                            variant={
-                                confirmButtonColorScheme ? "solid" : "default"
-                            }
-                            colorScheme={confirmButtonColorScheme}
-                            onClick={onConfirm}
-                        >
```
Comment (daniel-sm-yu) [inline src/components/modal/index.tsx:76]: @mkanoria I this PR undid https://github.com/uwblueprint/headsupguys/pull/29

**Pull request merged**